File: company/src/main/java/ru/scarlet/company/repository/ExpertiseRepository.java
package ru.scarlet.company.repository;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;
import ru.scarlet.company.entities.Expertise;

@Repository
public interface ExpertiseRepository extends JpaRepository<Expertise, Long> {
	Expertise findByName(String name);

}

File: company/src/main/java/ru/scarlet/company/repository/ExpertiseRepository.kt
package ru.scarlet.company.repository

import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import ru.scarlet.company.entities.Expertise

@Repository
interface ExpertiseRepository : JpaRepository<Expertise?, Long?> {
    fun findByName(name: String?): Expertise?
}